integration/e2ecortex/client.go

```diff
@@ -205,6 +205,10 @@ func (c *Client) GetAlertmanagerConfig(ctx context.Context) (*alertConfig.Config
 		return nil, ErrNotFound
 	}
 
+	if resp.StatusCode/100 != 2 {
+		return nil, fmt.Errorf("getting config failed with status %d and error %v", resp.StatusCode, string(body))
```

Comment:
Exposing the response body of the underlying storage is generally risky from a security perspective, because it may potentially leak sensitive data. I would discourage it.

Reply:
This is the test client though - is this still worth fixing?

Reply:
Definitely not. Sorry I was probably drunk 🤦